articles/reliability/reliability-fabric.md

```diff
@@ -171,7 +171,7 @@ When a major disaster renders the primary region unrecoverable, Microsoft Fabric
 
 The time it takes for failover to complete can vary, although it typically takes less than one hour. Once failover is complete, here's what you can expect:
 
-* **Fabric portal**: You can access the portal, and read operations such as browsing existing workspaces and items continue to work. All write operations, such as creating or modifying a workspace, are paused.
+* **Fabric portal**: You can access the portal, and read operations, such as browsing existing workspaces, task flows in workspaces, and items, continue to work. All write operations, such as creating or modifying a workspace, are paused.
```

Comment:
* **Fabric portal**: You can access the portal and read operations continue work. Read operations include browsing existing workspaces, workspace task flows, and items. All write operations, such as creating or modifying a workspace, are paused.

Reply:
@anaharris-ms Your edit was made after the PR was merged. Could you add this change in a new PR?